### ISSUE TITLE
can dask-labextension be  compatible with Jupyterlab-3.0.0 ?

### ISSUE THREAD

**Issue by stonebig** [dask/dask-labextension#149]
I understand that in Jupyterlab-3 cleaner world, we shall no more do "jupyter  labextension install dask-labextension".

**Comment by mrocklin** [comment]
Thank you for raising this issue @stonebig . I'm not sure what you mean.  Are you suggesting that we need to change our installation instructions?   If so, can you provide some background or suggestions?

**Comment by stonebig** [comment]
Jupyterlab-3 behavior with extensions is a bit blurry to me. As of now, i didn't succeed to make dask integration with Jupyterlab 3 working. 

So I was turning to the expert for clues, if you have already.

**Comment by jsignell** [comment]
I think we need to follow these steps: https://jupyterlab.readthedocs.io/en/latest/developer/extension_migration_2_3.html

**Comment by stonebig** [comment]
with jupyterlab-3.0.0rc15, I notice I can now do "as if" I was adding extension on jupyterlab2: bqplot, ipyleaflet, ipympl
The only exception is labextension that seems to set a constraint that may not be needed
````
jupyter labextension install dask-labextension
An error occured.
ValueError: The extension "dask-labextension" does not yet support the current version of JupyterLab.
````

**Comment by stonebig** [comment]
Maybe these constraints are relaxable ?


![image](https://user-images.githubusercontent.com/4312421/103095878-71ea6e00-4602-11eb-9e51-62ebe83473b2.png)

**Comment by ian-r-rose** [comment]
dask-labextension@5.0.0 has been released, which targets JupyterLab 3.0
